### PR TITLE
core docs

### PR DESCRIPTION
**Pull request by jgueldenstein**
## Proposed changes
Documentation for wolfgang core

## Necessary checks
- [x] Run `catkin build`
- [x] Write documentation
- ~[ ] Create issues for future work~
- [x] Test on your machine

### CODE REVIEW

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
++-----------+----------+-----------+-------------+
+
+Status LEDs
+===========
+
+The red round LED next to the manual switch indicated that the board is powered. If it is off even though the board is plugged in, the fuse might be blown.
+
+The green round LED next to it indicates if the motors and other devices on the Dynamixel bus are powered.
+
+The tree RGB LEDs are set by software.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:66]:
```suggestion
The three RGB LEDs are set by software.
```

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+Power over Ethernet
+===================
+
+The board features a Power over Ethernet (PoE) power sourcing equipment (PSE).
+To be more specific, it is a midspan, meaning it injects the power part of PoE onto the signal.
+It basically works like a PoE injector which are widely available but in a much nicer form factor for a robot.
+
+The powered device should be plugged into the RJ45 port pointing to the top and the other device into the port pointing to the bottom.
+
+PoE works with 48V which means you should not lick it.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:78]:
```suggestion
PoE works with 48V which means you should not lick it 😜.
```

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+Power Connectors and Power Source Selection
+===========================================
+
+There are two connectors meant to be used for soldering a connector (e.g. Tamiya or XT90) to connect a battery and power supply.
+
+They are located on the right above the Molex SPOX Mini connectors.
+They are labeled as VBAT+ and VBAT- for the battery and VEXT+ and VEXT- for the external supply. While technically they are treated the same,
+it is recommended to connect them in the correct order since only VEXT is measured and VBAT should be the same as the one connected to the balancer connector.
+The batteries balancer connector should be connected to P2. Be careful when soldering P2 since it needs to be oriented correctly.
+The GND pin is a the very bottom. If it is plugged in the other way, the Teensy will probably blow up.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:108]:
```suggestion
The GND pin is at the very bottom. If it is plugged in the other way, the Teensy will probably blow up 🤯.
```

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+This can be useful when changing batteries but keeping the robot powered on using a power supply.
+
+While some energy is lost over D2, this simple solution has proven very robust.
+
+
+Switching of power to Motors
+============================
+
+Power on the Dynamixel bus can be switched on and off using either the manual switch or through software.
+If the manual switch is to the right, the power if off. If it is to the right, the power is on (if the software agrees).
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:121]:
```suggestion
If the manual switch is to the right, the power is off. If it is to the right, the power is on (if the software agrees).
```

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+The manual switch is an override, meaning when it is off, the teensy can not enable it.
+If there is power on the Dynamixel bus the green 3mm round LED will be on. I
+f it is on when the switch is in the off position (to the right), the MOSFET Q1 is probably blown and needs replacement.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:124]:
```suggestion
If there is power on the Dynamixel bus the green round LED will be on.
If it is on when the switch is in the off position (to the right), the MOSFET Q1 is probably blown and needs replacement.
```

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+This may happen if there is a short circuit and the MOSFET is weaker than the fuse.
+
+When the power is turned off by software but is on by the switch,
+you can flip the switch off and on and the power should be back on provided the hardware hack for V1.0 is installed (see :ref:`Bodge Switch`).
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:128]:
```suggestion
When the power is turned off by software, but is on by the switch,
you can flip the switch off and on and the power should be back on, provided the hardware hack for V1.0 is installed (see :ref:`Bodge Switch`).
```

**Review by jaagut — DISMISSED** [review]
What a nice, comprehensive and well written guide!

**Comment by jgueldenstein** [comment]
Thanks for the suggestions, I will add a register table, wait with review please.

**Comment by jgueldenstein** [comment]
@jaagut ready for review

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 7]
```diff
@@ -0,0 +1,314 @@
...
+**VBAT_5**: Scale factor: (3.3 / 1024) * (1.8/(13.0+1.8))
+
+
+.. _Current Sensing:
+
+Current Sensing
+---------------
+
+Current is sensed using a Hall effect sensor (ACS712ELCTR-30A-T to be exact).
+It has to be scaled by the conversion factor from analog reading to voltage multiplied by the amperes per volt to get the actual current.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:310]: amperes per volt?

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst [hunk 8]
```diff
@@ -0,0 +1,314 @@
...
++--------+--------+---------------------------------------+--------+---------+---------+-------------+
+| 48     | 2      | :ref:`VBAT_5<Voltage Sensing>`        | r      |         | int16   | no          |
++--------+--------+---------------------------------------+--------+---------+---------+-------------+
+
+.. _DXL:
+
+DXL
+---
+
+**id**: Can be a value between 1 and 252. it is used to talk to the device over the Dynamixel bus.
```
Comment (jaagut) [inline bitbots_docs/docs/manual/hardware/wolfgang_core.rst:228]:
```suggestion
**id**: Can be a value between 1 and 252. It is used to talk to the device over the Dynamixel bus.
```

**Review by jaagut — APPROVED** [review]
Wow, thanks for all this detailed information. I have not double checked all these values. I trust you with thar one...

**Comment by jaagut** [comment]
What about the `.ods` file? Do you want to link this somewhere?

File: bitbots_docs/docs/manual/hardware/wolfgang_core.rst (hunk 7 above)
Comment (jgueldenstein) [reply bitbots_docs/docs/manual/hardware/wolfgang_core.rst:310]: yes, the voltage the sensor outputs goes up/down by 0.066 V per ampere that is flowing through it

**Comment by jgueldenstein** [comment]
> What about the `.ods` file? Do you want to link this somewhere?

No, it is only for generating the rst table automatically

**Review by jaagut — APPROVED** [review]

**Review by SammyRamone — APPROVED** [review]
maybe add a link to this documentation to the CORE git

**Pull request merged**